### PR TITLE
ios_user purge example

### PR DESCRIPTION
**Pull request by housysadm**
Added an example of purging all users except those listed and admin

##### ISSUE TYPE

 - Feature Pull Request
 - New Module Pull Request
 - Bugfix Pull Request
 - Docs Pull Request

##### ANSIBLE VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/modules/network/ios/ios_user.py [hunk 1]
```diff
@@ -114,7 +114,15 @@
 - name: remove all users except admin
   ios_user:
     purge: yes
-
+
```
Comment (gundalow) [inline lib/ansible/modules/network/ios/ios_user.py]: Please remove whitespace so we just have a blank line.

File: lib/ansible/modules/network/ios/ios_user.py [hunk 2]
```diff
@@ -114,7 +114,15 @@
...
-
+    
+- name: remove all users except admin and these listed users
+  ios_user:
+    aggregate:
+      - name: testuser1
+      - name: testuser2
+      - name: testuser3
+    purge: yes
+
```
Comment (gundalow) [inline lib/ansible/modules/network/ios/ios_user.py]: Please remove whitespace so we just have a blank line.

**Comment by housysadm** [comment]
I believe I have removed the extra spaces and pushed the revision but I forgot to comment when I did it. Please let me know if anything is still incorrect.

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
rebuild_merge

**Pull request merged**